src/input.css

```diff
@@ -1,3 +1,388 @@
+    @apply text-[46px] font-[600] leading-[110%] mb-[20px] text-[#1D2130];
+}
+.feature{
+    @apply text-[46px] font-[600] leading-[110%] mb-[20px] text-[#1D2130] text-center; 
+}
+.information__title {
+    @apply text-2xl
+}
+.hero__text{
+    @apply mb-[30px];
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 150%;
+    text-align: center;
+    color: #1D2130;
+    opacity: 0.87;
+}
+.form, .hero__form{
+    @apply bg-[white] w-fit m-auto p-[4px] rounded-[12px] ;
+}
+.form__input, .hero__input{
+    @apply text-black outline-none w-[400px] placeholder:text-[#1D2130] text-[16px] pl-[12px];
+}
+.navbar__btn, .btn, .hero__btn, .additional-info__btn, .product-header-description__btn{
+    @apply bg-[#1D2130] text-white py-2 px-7 rounded-lg
+}
+.additional-info__btn--raised{
+    @apply mt-10
+}
+.btn--bg{
+    @apply text-[black] bg-white
+}
+.btn--white, .product-header-description__btn--white-bordered{
+    @apply bg-white text-black border-[3px];
+}
+.hero__img{
+    @apply m-auto mt-[100px] 
+} 
+.footer-ascend__form{
+    @apply bg-[#2B2E3C] w-fit m-auto  rounded-[12px] ;
+}
+.footer-ascend__input{
+    @apply text-black outline-none bg-transparent p-[12px] placeholder:text-[white] placeholder:opacity-[0.5] text-[16px] pl-[12px];
+}
+.quality-section{
+    @apply text-center mt-[100px];
+}
+.title-sm, .quality-section__title--blue, .information__title--small, .sales__small-title, .markerting__small-title, .time-tracker__small-title, .lead-generation__small-title, .remote-team__small-title, .remote-team__small-title{
+    @apply text-[24px] font-semibold bg-clip-text text-transparent bg-gradient-to-r from-[#373FFF] to-[#3ACAF8] mb-[20px]; 
+}
+.feature-section{
+    @apply grid grid-cols-3 m-auto w-fit bg-[#E0E4FC] p-[40px] pt-[50px] pb-[50px] gap-[40px] rounded-[24px] mr-[5%] ml-[5%] mt-[50px];
+}
+.feature-section__img{
+    @apply w-[66.44px] mb-[20px];
+}
+.feature-section__title{
+    @apply text-[32px] font-[700] leading-[120%] mt-[10px] mb-[20px];
+}
+.feature-section__description, .sales__description{
+    @apply font-[400] text-[16px] opacity-[0.87] mt-[10px];
+} 
+.information, .sales, .markerting, .time-tracker, .lead-generation, .remote-team{
+    @apply mr-[5%] ml-[5%]  mt-[120px] gap-[40px] grid grid-cols-2 mb-[100px];
+}
+.information__desc, .markerting__description, .time-tracker__description, .lead-generation__description{
+    @apply text-[20px] font-[400] leading-[150%];
+}
+.information__item--right, .sales__item--right, .markerting__image-container, .time-tracker__image-container{
+    @apply m-auto
+}
+.information__details, .sales__details{
+    @apply flex gap-[20px];
+}
+.information__icon, .sales__icon{
+    @apply bg-[#FFACAC] h-[48px] w-[48px] mt-auto mb-auto p-[17px] rounded-[50%];
+}
+.information__desc{
```

Comment:
use information__description instead of information__desc

Reply:
I changed it